recipes/codem/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "codem" %}
+{% set version = "0.23.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/codem-{{ version }}.tar.gz
+  sha256: 21f609590b11d6e552ecf62c7872bc200f3e6b2e191eb0527e9a55ee512f8950
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - codem = codem.main:main
+    - vcd = vcd.main:main 
+
+requirements:
+  build:
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
```

Comment:
I think this one can be noarch. These are probably not needed. They are used to link only and I did not find any compiled extension here.

Reply:
Do you think we can do away with `xorg-libxfixes  # [linux]` as well? If so, we should be able to remain noarch.

Reply:
Yep. All you need is `mesa-libGL` in the `yum_requirements.txt`. (Just tested it locally.)